### ISSUE TITLE
Text align options not working 

### ISSUE THREAD

**Issue by adrienpicard** [verbb/vizy#63]
**Description**
Text align options no longer work in `1.0.5`

**Steps to reproduce**
1. Update to Vizy `1.0.5`
2. Try changing text alignment

**Additional info**
- Plugin version: 1.0.5
- Craft version: 3.7.7
- Multi-site: No

**Additional context**
Here is the field configuration:

`{
  "buttons": ["align-left", "align-right"],
  "toolbarFixed": true
}`

Reverting to `1.0.4` fixes the issue.

**Comment by engram-design** [comment]
Bah - looks like [tiptap](https://github.com/ueberdosis/tiptap/commit/b4e41afd1798e7eba4ccb0804be1ed2473c1f4d2) has gone ahead and removed the default nodes alignment options get applied on. 

```
types | Array | [] | A list of nodes where the text align attribute should be applied to. Usually something like ['heading', 'paragraph'].
```

This is neat for future configuration, but they've made this a breaking change setting this to `[]` (none) by default.

Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.5",
  "...": "..."
}
```

Then run `composer update`.

**Comment by adrienpicard** [comment]
Cheers @engram-design, that was quick!

**Comment by engram-design** [comment]
Fixed in [1.0.6](https://github.com/verbb/vizy/releases/tag/1.0.6)
